util.go

```diff
@@ -33,7 +34,9 @@ const (
 )
 
 func init() {
-	seed.Init()
+	if _, err := seed.Init(); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
log.Fatal calls os.Exit, which is an extremely problematic side-effect.
Swallow the error instead.